src/environment/ssDaylight.lua

```diff
@@ -11,7 +11,7 @@ ssDaylight = {}
 g_seasons.daylight = ssDaylight
 
 function ssDaylight:load(savegame, key)
-    self.latitude = Utils.clamp(ssXMLUtil.getXMLFloat(savegame, key .. ".environment.latitude", 51.9), 0, 72)
+    self.latitude = Utils.clamp(ssXMLUtil.getXMLFloat(savegame, key .. ".environment.latitude", 47.4), 0, 72)
```

Comment:
Why did you change this from the default we have? The latitude we had is from the same area we have growth and weather data from.